src/dependencytracker.js

```diff
@@ -252,6 +255,30 @@ class DependencyTracker {
         }
       },
     });
+
+    let task = await this.Task.load({taskId: taskId});
```

Comment:
I suggest checking the call site for `resolveTask` I'm sure it already has the task entity... so we can probably just take taskGroupId as an other option....

If there are places where it isn't present we can take it as an optional argument... we only do the Task lookup here because we need the taskGroupId, right?